src/library/templates/library/edit_customization.html

```diff
@@ -103,5 +126,146 @@ <h2>Edit {{ customization.title }} {% if is_new == 'true' %}(New){% endif %}</h2
        $input.val($input.val() + text);
        $input.focus();
     });
+
+    $('#add-word-button').click(function (e) {
+        addVocabularyWord(e);
+    });
+
+    $('#id_word_to_add').keypress(function (e) {
+        var keycode = (e.keyCode ? event.keyCode : event.which);
+        if (keycode == '13') {
+            addVocabularyWord(e);
+        }
+    });
+
+    $('.delete-word-button').click(function (e) {
+        deleteVocabularyWord(e);
+    });
+
+    function deleteVocabularyWord (e) {
+        e.preventDefault();
+        var parentDiv = $(e.currentTarget).parent();
+        var wordToDelete= $(e.currentTarget).attr('id').replace(/^delete_/,'').replace('_', ' ');
```

Comment:
You can avoid having to do string substitutions like this by using a data attribute, eg data-word="{{word}}",  wordToDelete=$(e.currentTarget).data('word')

Reply:
Great idea!  I've used the `@data` attribute in the past, but it didn't occur to me this time, nor for this purpose.  But it's definitely cleaner.